standards/WIT-SVID.md

```diff
@@ -0,0 +1,151 @@
+
+Typically, the `jti` will be an opaque randomly generated value of sufficient entropy as to make the chance of collision negligible.
+
+Primarily, this claim enables distinguishing one or more WIT-SVIDs that contain the same SPIFFE ID for the purposes of auditing. For example, if a validator of a WIT-SVID records the JTI within an audit log event, this audit log event can be correlated with the one emitted by the issuer which allows the lineage of the credential to be ascertained.
+
+Due to the nature of how this claim uniquely identifies the WIT-SVID, it could be leveraged for revocation of an individual WIT-SVID. There are no mechanisms defined within SPIFFE for the propagation of WIT-SVID revocations and this is considered out of the scope of the specification.
+
+### 3.2. Additional Claims
+
+It is permitted for an implementation to include additional claims not specified in this document or the upstream document.
+
+## 4. Token Signing and Validation
+
+## 5. Token Presentation
+
+This section describes the manner in which a WIT-SVID may be presented from one workload to another for the purposes of authentication.
+
+The WIT-SVID MUST always be presented by the workload with proof of possession of the key-pair contained within the `cnf`. In other words, the WIT-SVID MUST NOT be presented as a bearer token.
+
+WIMSE defines protocols for presentation of the WIT. It is recommended that that an implementor use one of these defined protocols, however, it is not required. At the time of writing, there are two protocols:
+
+- [WIMSE Workload Proof Token][4]
+- [WIMSE Workload-To-Workload Authentication with HTTP Signatures][5]
+
+## 6. Representation in the SPIFFE Bundle
+
+This section describes how the WIT-SVID signing keys are published to and consumed from a SPIFFE bundle. Please see the [SPIFFE Trust Domain and Bundle](SPIFFE_Trust_Domain_and_Bundle.md) specification for more information about SPIFFE bundles.
+
+### 6.1 Publishing SPIFFE Bundle Elements
+
+WIT-SVID signing keys for a given trust domain are represented in the SPIFFE bundle as [RFC 7517][3]-compliant JWK entries, one entry per signing key.
+
+The `use` parameter of the JWK entry MUST be set to `wit-svid`. Additionally, the `kid` parameter of each JWK entry must be set.
+
+### 6.2 Consuming SPIFFE Bundle Elements
+
+SPIFFE bundles may contain JWK entries for many different SVID types. Implementations MUST extract the WIT-SVID specification keys before using them for validation purposes. Entries representing WIT-SVID signing keys can be identified by the value of their `use` parameter, which must be `wit-svid`. If there are no entries with the `wit-svid` use parameter, then the trust domain that the bundle represents does not support WIT-SVID.
```

Comment:
```
If there are no entries with the `wit-svid` use parameter, then the trust domain that the bundle represents does not support WIT-SVID.
```
Does this make a nice indicator to detect of your peer TD supports WIT-SVID 🤔 I wonder if there's implications for SPIFFE Federation in general.

Reply:
I don't think federation is impacted beyond what has been written so far (i.e., that new key uses will now be present if supported).